process/smartContract/process.go

```diff
@@ -300,6 +304,9 @@ func (sc *scProcessor) GasScheduleChange(gasSchedule map[string]map[string]uint6
 	}
 
 	sc.builtInGasCosts = builtInFuncCost
+	if sc.flagFixAsyncCallBackArgumentsParser.IsSet() {
+		sc.builtInGasCosts[core.BuiltInFunctionMultiESDTNFTTransfer] = builtInFuncCost["ESDTNFTMultiTransfer"]
```

Comment:
this will be pretty hard to keep track. Why not create a new gas map that will be activated exactly when the flagFixAsyncCallBackArgumentsParser is set?

Reply:
Here we have an issue of naming in gasSchedule vs builtInFunctions name and sc.bultInGasCosts[function] returned 0. Which was wrong. This is the case of different string constants between core.BuiltInFunctionMultiESDTNFTTransfer and the one from gas schedule. If I add it into the gasSchedule it will not be backward compatible.

Reply:
It will be, only in a v7 of the gas map that will be active in the same epoch with the fix

Reply:
Understood the problem, I do not have a better way to fix this since all variants will involve a flag like this